lms/services/jstor.py

```diff
@@ -97,7 +98,19 @@ def thumbnail(self, article_id: str):
         :raise ExternalRequestError: If the response doesn't look like a valid `data:` URI
         """
         doi = self._doi_from_article_id(article_id)
-        data_uri = self._api_request(format_uri("/thumbnail/{}", doi)).text
+        params = {
+            # `offset` specifies the page number. The default value of 0 returns
```

Comment:
This works here and it's well documented but I wonder if something like "width" would make more sense to be forwarded from the frontend?

Os is there a concern it could be manipulated and request huge images?

Reply:
> This works here and it's well documented but I wonder if something like "width" would make more sense to be forwarded from the frontend?

Yes, logically it would make sense for the frontend to specify the image size it wants. It has been avoided so far just to reduce the amount of plumbing involved.

> Os is there a concern it could be manipulated and request huge images?

The LMS API is currently only available to our frontend, so API abuse isn't a major concern, but it probably would be a good idea to put min/max constraints on the width value on the backend.